src/indexer/index.ts

```diff
@@ -190,14 +190,17 @@ export const isApplicationOpted = async (
 	application_id: number
 ): Promise<boolean> => {
 	try {
-		const optedApplicationInfo = await AlgoSigner.algod({
-			ledger: CHAIN_NAME,
-			path: `/v2/accounts/${address}/applications/${application_id}`,
-		});
-		if (optedApplicationInfo && optedApplicationInfo["app-local-state"]) {
-			return optedApplicationInfo["app-local-state"].id === application_id;
+		let isAppOpted = false;
+		const applicationInfo = await indexerClient.lookupAccountByID(address).do();
```

Comment:
Check here If using `indexerClient` gives same response as `AlgoSigner.algod`. I think it should be `algodClient`

Reply:
It gives the same response, its just that the endpoints are different, I updated it so as to minimise the use of algod since most of the data can be fetched from the indexer itself

Reply:
ok